widgets/google-map/google-map.php

```diff
@@ -596,12 +630,30 @@ function get_template_variables( $instance, $args ) {
 
 	function get_less_variables( $instance ) {
 		$global_settings = $this->get_global_settings();
-
-		return array(
+		$less_variables = array(
 			'height' => $instance['settings']['height'] . 'px',
 			'map_consent' => ! empty( $global_settings['map_consent'] ),
 			'responsive_breakpoint' => ! empty( $global_settings['responsive_breakpoint'] ) ? $global_settings['responsive_breakpoint'] : '780',
 		);
+
+		// Map Content Button styling.
+		if ( ! empty( $global_settings['map_consent_design']['button']['color'] ) ) {
```

Comment:
This could be a loop to make things a little more maintainable. Something like:

```
if(!empty($global_settings['map_consent_design']['button']) && is_array($global_settings['map_consent_design']['button'])) {
    foreach($global_settings['map_consent_design']['button'] as $style => $value){
        if( ! empty($value) ) $less_variables['map_consent_notice_button_'.$style] = $value;
    }
}